docs/guides/latest/advance-use-case/instant-backup.md

```diff
@@ -0,0 +1,378 @@
+# Instant Backup
+
+This guide will show you how to trigger a backup volumes instantly using Stash.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [Minikube](https://github.com/kubernetes/minikube).
+
+- Install `Stash` in your cluster following the steps [here](https://appscode.com/products/stash/0.8.3/setup/install/).
+
+- You should be familiar with the following `Stash` concepts:
+  - [BackupConfiguration](/docs/concepts/crds/backupconfiguration.md/)
+  - [BackupSession](/docs/concepts/crds/backupsession.md/)
+  - [Repository](/docs/concepts/crds/repository.md/)
+  
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```console
+$ kubectl create ns demo
+namespace/demo created
+```
+
+>**Note:** YAML files used in this tutorial are stored in  [docs/examples/guides/latest/advance-use-case/instant-backup](/docs/examples/guides/latest/advance-use-case/instant-backup) directory of [stashed/stash](https://github.com/stashed/stash) repository.
+
+## Backup Configure
+
+Here, we are going to backup the data volumes of Deployment. At first, we will deploy a Deployment with a PVC and generate some sample data in it. Then we will take instant backup of these PVC using Stash.
+
+**Deploy Deployment:**
+
+We will deploy a Deployment with PVC. This Deployment will automatically generate sample data in `/source/data` directory where we have mounted the PVC.
+
+Below is the YAML of the Deployment and PVC that we are going to create,
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: source-data
+  namespace: demo
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: standard
+  resources:
+    requests:
+      storage: 2Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-demo
+  name: stash-demo
+  namespace: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stash-demo
+  template:
+    metadata:
+      labels:
+        app: stash-demo
+      name: busybox
+    spec:
+      containers:
+      - args: ["echo sample_data > /source/data/data.txt && sleep 3000"]
+        command: ["/bin/sh", "-c"]
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /source/data
+          name: source-data
+      restartPolicy: Always
+      volumes:
+      - name: source-data
+        persistentVolumeClaim:
+          claimName: source-data
+```
+
```

Comment:
The above Deployment will automatically create a `data.txt` file in `/source/data` directory and write some sample data in it.